### DISCUSSION TITLE
Unable to buy land when not using money

### DISCUSSION THREAD

**Discussion by Broxzier** [OpenRCT2/OpenRCT2#15717] [Feature requests]
In scenarios where you can extend your park, you are unable to buy the land normally, and have to enable sandbox mode with it, which in turn has no restrictions at all. To keep the restrictions that were set when the scenario was designed and be able to buy land, you now have to enable using money, open the window to buy land from the park window, then buy the land, and disable money again.

I do see that 'buying' land is kind of weird when you play without money anyway, though keeping the restrictions adds to the gameplay in my experience..

Some ways of resolving it that I can think of, and like in order from good to bad:
- Display the land ownership button, and make it open the menu as usual, but the land to buy is free. This keeps the restrictions that were set when the scenario was designed.
- Display the land ownership button, but with an option to own all land that is for sale.
- Display the land ownership button, but make it open map with sandbox mode enabled when not using money, and disable it again when closing the window.
- When designing a scenario and disabling the use of money, prevent the map from being saved where there is land in it that can be bought.

One thing I also noticed is that when you disable money while buying land, you still pay for the land.

**Comment by IntelOrca** [comment]
In my opinion, having buyable land in a no money scenario is pointless. The player can immediately just purchase all the land as soon as they start the scenario.
- If the player doesn't want to do that because they want to shape their park perimeter nicely... they might as well use sandbox mode.
- Construction rights can already be set to be pre-owned in a scenario, so that ability is not lost.

So in my opinion I would go for the latter option, although rather than prevent them from saving - just give a warning message that says that the buyable land will be automatically bought when the scenario is saved. Additionally, starting a scenario that has no-money could also immediately buy the land rights. Probably not a good idea to do this though if the player changes the money setting during the game.
